lblod/ember-rdfa-editor-lblod-plugins
# Reduce number of dummy routes

### Overview
This PR reduces the number of decision routes:
- The `editable-nodes-besluit` and `besluit-sample` routes have been merged into `besluit-sample`. The `besluit-sample` route now contains a toggle to enable/disable the option to edit nodes.
- The `editable-nodes-regulatory-statement` and `regulatory-statement-sample` routes have been merged into `regulatory-statement-sample`. The `regulatory-statement-sample` route now contains a toggle to enable/disable the option to edit nodes.
- The `editable-node` dummy route has been removed as it has become obsolete.

### How to test/reproduce
- `pnpm start`
- ensure that in both dummy routes you have the option to edit the different nodes
